schema-bridge.graphql

```diff
@@ -55,6 +66,30 @@ type Token @entity {
   lastPriceBlockNumber: BigInt
 }
 
+# CrosschainToken is the representation of a token from a different chain.
+# For example, if you bridge USDC from Ethereum to Polygon, the token on Polygon is a CrosschainToken.
+# CrosschainToken might or might not be canonical. It refers to the counterpart of the token on the other chain.
+# If the token is wrapped on the current chain, CrosschainToken will be canonical on the opposite chain; and viceversa.
```

Comment:
Is this always true? If pools are liquidity <> liquidity, could the tokens on each side not be wrapped?

Reply:
Oh, yeah, not really. If one is wrapped the other one will be canonical for sure. But one being canonical doesn’t imply the other one being wrapped. The vice versa is wrong there. Added more clarity there.